capstone_project/src/main/webapp/pages/mapsTemplate.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Maps Feature</title>
+    <link rel="stylesheet" href="../styles/maps_style.css">
+    <link href="https://fonts.googleapis.com/css?family=Quicksand:100,200,300" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=CAPSTONE_API_KEY"></script>
+    <script src=" ../scripts/mapsScriptTemplate.js"></script>
+  </head>
+  <body onload="createMap();">
+    <div id="content">
+      <h1>Maps Feature</h1>
+      <p>This is the maps feature of our capstone project.</p>
+      <br>
+      <div id="map"></div>
+      <br>
+      <h3>Don't see your college?</h3>
+      </p>Submit your school's info below and it'll appear on the map!</p>
+      <form action="/school-data" method="POST">
+        <p>School Name:</p>
+        <!-- <textarea name= "name-input" required></textarea> -->
+        <input name= "name-input" required>
```

Comment:
nit: remove the blank space between = and "
same for the others below

Reply:
@Leoism fixed!